docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -519,3 +519,67 @@ Standard ONNX\* operators:
 | Upsample | No |
 | Where | No |
 | Xor | No |
+
+
+## PaddlePaddle\* Supported Operators
+
+Standard PaddlePaddle(paddlepaddle>=2.1)\* Operators:
+
+| Operator Name in PaddlePaddle\*| Limitations|
+| :----------| :----------|
+| adpative_pool2d | Only support 'NCHW' input data_layout |
+| arg_max | No |
+| assign_value | No |
+| batch_norm | No |
+| bilinear_interp | Only supports the same data_layout |
+| bilinear_interp_v2 | Only supports the same data_layout |
```

Comment:
This same should be NCHW

Reply:
In ngraph code, I write `TODO support different data_layout`. It means that later we should support data layouts other than default NCHW.   Not the SAME layout...

Reply:
```suggestion
| bilinear_interp_v2 | Only supports the NCHW data_layout |
```